lazy_tensor_core/lazy_tensor_core/csrc/tensor_aten_ops.cpp

```diff
@@ -272,14 +279,15 @@ torch::lazy::LazyTensorPtr squeeze(const torch::lazy::LazyTensorPtr& input, int6
 }
 
 void squeeze_(torch::lazy::LazyTensorPtr& input) {
-  input->SetIrValue(
-      torch::lazy::MakeNode<ir::ops::Squeeze>(input->GetIrValue(), -1));
+  input->SetIrValue(torch::lazy::ReuseOrMakeNode<ir::ops::Squeeze>(
```

Comment:
its up to you, but i have been deleting the `torch::lazy` prefix in a lot of code once it finally landed in a place where everything was in that namespace.  If you are s/replacing anyway, might be worth trimming it but no big deal to keep it